### PR TITLE
update release scripts to run tests before publishing an image

### PR DESCRIPTION
**Pull request by eljohnson92**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests
- [ ] adds or updates e2e tests

### CODE REVIEW

File: .github/workflows/build_test_ci.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ on:
     paths-ignore:
       - '**/**.md'
       - 'docs/**'
+  workflow_dispatch:
```
Comment (komer3) [inline .github/workflows/build_test_ci.yml:16]: Question: What is this for? Why does it have no inputs?

Comment (eljohnson92) [reply .github/workflows/build_test_ci.yml:16]: This allows us to trigger a build from the actions tab as well, it doesn't need any specific inputs

Comment (komer3) [reply .github/workflows/build_test_ci.yml:16]: Oh super cool!

**Review by komer3 — DISMISSED** [review]
LGTM!

**Pull request merged**